requirements-test.txt

```diff
@@ -7,4 +7,5 @@ pytest-cov==2.5.1
 pytest-parallel==0.0.2
 pymongo==3.7.1
 pytest-mongodb==2.1.2
-black==18.6b3
\ No newline at end of file
+black==18.6b3
```

Comment:
Any way to clean up these two lines?

Reply:
Sorry - what is it you'd like cleaned up?

Reply:
Hey's just asking why there is a `diff` here. As in, what did you change?

Reply:
Ah, this was just adding black as a dependency for testing.  Going to mark this as resolved, as there's nothing to do here.